src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.tasks.Task;
+import duke.tasks.TaskType;
+
+/**
+ * Command sub-type to define adding Tasks
+ */
+public class AddCommand extends Command {
+
+    private final TaskType taskType;
+
+    /**
+     * Creates AddCommand object.
+     *
+     * @param attributes input attributes from user
+     * @param taskType TaskType of the Task to add
+     */
+    public AddCommand(String attributes, TaskType taskType) {
+        this.attributes = attributes;
+        this.taskType = taskType;
+    }
+
+    @Override
```

Comment:
Perhaps we can add a Javadoc comment here? Or use `{@inheritDoc}` to inherit parent Javadocs. Think this goes for other overriden methods as well!